content/en/monitors/incident_management/analytics.md

```diff
@@ -2,10 +2,6 @@
 title: Incident Management Analytics
 kind: documentation
 description: Track and analyze aggregated incident management statistics in Dashboards and Notebooks
-further_reading:
-- link: "dashboards/querying/#incident-management-analytics"
-  tag: "Documentation"
-  text: "Incident Management Analytics"
```

Comment:
Since you are eliminating the entire Further Reading section, you also need to remove lines 50 & 52.